templates/docker-compose.template

```diff
@@ -125,8 +115,17 @@ services:
 
```

Comment:
in the readme text above, you have removed the DBUS session text.

But the container is still here.

Do we need the container still?

Reply:
Not exactly removed. I remember I've removed dbus container and merged it into hm-diag. But then we needed dbus-session (system dbus is different). So we had introduced this one is is a dependency right now. I'm not sure how much we need this, we can discuss that with team, but right now it's better to leave it as it is.

Reply:
the only reason i was asking is because the info about the container was removed from the readme. but if we arent removing the container then we should add it back to readme I guess until we actually remove it (if at all)

I think we were using the dbus container for the miner container to communicate with gateway-config container for the onboardings.

do we not need that any more?

Reply:
The removal was done a long time ago probably, back then when I had removed that. But somehow we added it back, and not sure why.

Let's keep this as it doesn't create any issues, but later discuss its necessity and remove it if OK. Deal? :smile: